docs/index.md

```diff
@@ -3,174 +3,148 @@
 page_title: "ocm Provider"
 subcategory: ""
 description: |-
+  
 ---
-# OCM Provider
+<a href="https://redhat.com">
+    <img src=".github/Logo_Red_Hat.png" alt="RedHat logo" title="RedHat" align="right" height="50" />
+</a>
+
+# Red Hat OCM Provider
+
+> Please note that this Terraform provider and its modules are open source and will continue to iterate features, gradually maturing this code.
+> If you encounter any issues, please report them in this repo.
```

Comment:
Need here new line (In the marked down preview its continue in the previous line)